WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -142,8 +163,10 @@ extension PointOfSaleAggregateModel {
         cart.removeAll()
     }
 
-    func submitCart() {
+    @MainActor
+    func submitCart() async {
         orderStage = .finalizing
+        await checkOut()
     }
```

Comment:
I'm planning on replacing this with a single `checkOut` function that does this.